docs/root/configuration/best_practices/edge.rst

```diff
@@ -1,5 +1,91 @@
+.. _best_practices_edge:
+
 Configuring Envoy for the edge
 ==============================
 
-This document describes best practices for configuring Envoy for the edge. It is currently a
-work-in-progress
+Envoy is a production-ready edge proxy, and performs well in that role, however,
```

Comment:
I think this first sentence either needs citations, e.g. links to folks who are blogging about this, or it needs to be toned down a bit. We definitely know of a number of folks using Envoy at the edge, but in terms of the number of QPS and applications Envoy has been exposed to at the edge vs. a number of other well known proxies.. I think these are early days.

Reply:
Whilst not particularly keen on writing a blog about it on my current schedule, we're running Envoy at what I would think is a decent enough scale to claim production-readiness as an edge proxy... Not sure how exactly I would qualify this, by bytes transferred daily/weekly? Something else?

Reply:
Although it is early days compared to some other proxies, given the number of well known enterprises using Envoy at the edge, I think we can say that it's a production ready edge proxy (that is not to say that things cannot be improved, though I would argue that some of those improvements are above and beyond what other publicly available solutions provide). I don't think we need citations in our own docs; there is some amount of leeway granted. :)

With that said, if we want to change the language slightly no huge objection, I just don't want to change the message entirely.

Reply:
I guess I have an objection from a critical language reading perspective. The aside "and performs well in that role" does not add much value to the message being conveyed and can be interpreted as hyperbole. The reason I mention citations is that if there was some place that somebody was saying "wow, look at this amazing Envoy edge proxy, its blows our minds", then that would justify this language, otherwise I would drop this part of the sentence.

Reply:
Sure, sounds good.